SECURITY.md

```diff
@@ -2,10 +2,20 @@
 
 ## Reporting a Vulnerability
 
-To report security issues please send an e-mail to [security@lightprotocol.com](mailto:security@lightprotocol.com).
+We appreciate any vulnerability found and responsibly disclosed.
```

Comment:
maybe this is a good place to mention terms.

Reply:
See here for guidelines on responsible disclosure. Only responsibly disclosed vulnerabilities are eligible for a bounty.